README.md

```diff
@@ -12,7 +14,7 @@ When creating an application load balancer, Amazon creates a hostname like `publ
 
 The `lb` module therefore creates a Route 53 alias record like `public.some-fully-qualified-domain-name` that can be used by downstream consumers (like users of the `ecs-service` module) as a symbolic reference to the load balancer, and which can be easily reprovisioned if the load balancer needs to be spun down and relaunched.
 
-Use the source `git@github.com:cites-illinois/as-aws-modules//lb` when creating a load balancer with one or more Route 53 aliases. For more information, see the [manual page for the **lb** module](https://github.com/cites-illinois/as-aws-modules/blob/master/lb/README.md).
+Use the source `git@github.com:techservicesillinois/terraform-aws-alias-lb-/b` when creating a load balancer with one or more Route 53 aliases. For more information, see the [manual page for the **lb** module](https://github.com/techservicesillinois/terraform-aws-lb/README.md).
```

Comment:
Change `git@github.com:techservicesillinois/terraform-aws-alias-lb-/b` to `git@github.com:techservicesillinois/terraform-aws-alias-lb`

Reply:
In addition fix the broken link to the lb module from:

https://github.com/techservicesillinois/terraform-aws-lb/README.md

to:

https://github.com/techservicesillinois/terraform-aws-lb

or possibly:

https://github.com/techservicesillinois/terraform-aws-lb/blob/master/README.md

But I prefer the former.